code/modules/clothing/masks/gasmask.dm

```diff
@@ -190,72 +190,213 @@
 	name = "security gas mask"
 	desc = "A standard issue Security gas mask with integrated 'Compli-o-nator 3000' device, plays over a dozen pre-recorded compliance phrases designed to get scumbags to stand still whilst you taze them. Do not tamper with the device."
 	icon_state = "sechailer"
-	var/aggressiveness = 2
+	var/phrase = 1
+	var/aggressiveness = 1
 	var/safety = 1
-	actions_types = list(/datum/action/item_action/halt, /datum/action/item_action/adjust)
+	actions_types = list(/datum/action/item_action/halt, /datum/action/item_action/adjust, /datum/action/item_action/selectphrase)
 
 /obj/item/clothing/mask/gas/sechailer/hos
 	name = "\improper HOS SWAT mask"
 	desc = "A close-fitting tactical mask with an especially aggressive Compli-o-nator 3000. It has a tan stripe."
 	icon_state = "hosmask"
 	aggressiveness = 3
-	actions_types = list(/datum/action/item_action/halt)
+	phrase = 12
+	actions_types = list(/datum/action/item_action/halt, /datum/action/item_action/selectphrase)
 
 /obj/item/clothing/mask/gas/sechailer/warden
 	name = "\improper Warden SWAT mask"
 	desc = "A close-fitting tactical mask with an especially aggressive Compli-o-nator 3000. It has a blue stripe."
 	icon_state = "wardenmask"
 	aggressiveness = 3
-	actions_types = list(/datum/action/item_action/halt)
+	phrase = 12
+	actions_types = list(/datum/action/item_action/halt, /datum/action/item_action/selectphrase)
 
 
 /obj/item/clothing/mask/gas/sechailer/swat
 	name = "\improper SWAT mask"
 	desc = "A close-fitting tactical mask with an especially aggressive Compli-o-nator 3000."
 	icon_state = "officermask"
 	aggressiveness = 3
-	actions_types = list(/datum/action/item_action/halt)
+	phrase = 12
+	actions_types = list(/datum/action/item_action/halt, /datum/action/item_action/selectphrase)
 
 /obj/item/clothing/mask/gas/sechailer/blue
 	name = "\improper blue SWAT mask"
 	desc = "A neon blue swat mask, used for demoralizing Greytide in the wild."
 	icon_state = "blue_sechailer"
 	item_state = "blue_sechailer"
 	aggressiveness = 3
-	actions_types = list(/datum/action/item_action/halt)
+	phrase = 12
+	actions_types = list(/datum/action/item_action/halt, /datum/action/item_action/selectphrase)
 
 /obj/item/clothing/mask/gas/sechailer/cyborg
 	name = "security hailer"
 	desc = "A set of recognizable pre-recorded messages for cyborgs to use when apprehending criminals."
 	icon = 'icons/obj/device.dmi'
 	icon_state = "taperecorder_idle"
-	aggressiveness = 1 //Borgs are nicecurity!
-	actions_types = list(/datum/action/item_action/halt)
+	actions_types = list(/datum/action/item_action/halt, /datum/action/item_action/selectphrase)
 
 /obj/item/clothing/mask/gas/sechailer/ui_action_click(mob/user, actiontype)
 	if(actiontype == /datum/action/item_action/halt)
 		halt()
-	else
+	if(actiontype == /datum/action/item_action/adjust)
 		adjustmask(user)
+	if(actiontype == /datum/action/item_action/selectphrase)

```

Comment:
The lines `if(actiontype == /datum/action/item_action/adjust)` and `if(actiontype == /datum/action/item_action/selectphrase)` should be changed to `else if` rather than just `if`, as otherwise you'll needlessly check these despite the fact that they're never going to evaluate to `true`.